src/javascripts/todo-item.js

```diff
@@ -87,4 +87,4 @@ class TodoItem {
   }
 }
 
-export default TodoItem
+return TodoItem
```

Comment:
@Prerna1993 Why is this module returning the class instead of exporting it by default?